### PR TITLE
feat(content_service): Add a generic content storage service with mul…

### PR DESCRIPTION
**Pull request by fhuaulme**
…tiple storage support.

* Initial commit for content service.

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Tests for the changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/131/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/131/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/137/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/137/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/138/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/138/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/140/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/140/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/148/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/148/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/152/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/152/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/154/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/154/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/155/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/155/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/159/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/159/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/164/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/164/)

File: daikon-content-service/s3-content-service/src/main/java/org/talend/daikon/content/s3/LocationUtils.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package org.talend.daikon.content.s3;
+
+/**
+ * Utility class for processing location in order to algin them with S3's internals.
```
Comment (sgandon) [inline daikon-content-service/s3-content-service/src/main/java/org/talend/daikon/content/s3/LocationUtils.java]: typo here *algin* instead of *align*

**Comment by sgandon** [comment]
Could you also fix codacy issues please ?

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/172/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/172/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/173/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/173/)

**Pull request merged**